frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/SmartScalarSettingsWidgets.styled.tsx

```diff
@@ -13,29 +13,24 @@ export const ComparisonList = styled.ul`
   }
 `;
 
-export const AddComparisonButton = styled(Button)<ButtonProps>`
+export const AddComparisonButton = styled((props: ButtonProps) => (
+  <Button {...props} variant={props.variant ?? "subtle"} />
+))`
   align-self: flex-start;
   padding: 0;
 `;
 
-AddComparisonButton.defaultProps = {
-  variant: "subtle",
-};
-
 type ComparisonPickerSecondaryTextProps = TextProps &
   HTMLAttributes<HTMLSpanElement> & {
     component?: "span";
   };
 
 export const ComparisonPickerSecondaryText = styled(
-  Text,
+  (props: ComparisonPickerSecondaryTextProps) => (
+    <Text {...props} component={props.component ?? "span"} />
+  ),
 )<ComparisonPickerSecondaryTextProps>``;
 
-ComparisonPickerSecondaryText.defaultProps = {
-  component: "span",
-  color: "text.0",
```

Comment:
I think this is wrong, don't think we support `text.0`. I removed this as I think it is invalid.